bin/initoverlayfs-install

```diff
@@ -37,6 +37,30 @@ exec_erofs() {
     popd
     rm -f "${INITRAMFS_DIR}/initoverlayfs-$kver.img"
     mkfs.erofs $erofs_compression "${INITRAMFS_DIR}/initoverlayfs-$kver.img" ${INITRAMFS_DUMP_DIR}
+
+	generate_dm_verity_hash "${INITRAMFS_DIR}/initoverlayfs-$kver.img" "${INITRAMFS_DIR}/verity_table.img" $INITOVERLAYFS_CONF
```

Comment:
just wondering if there are extra spaces in the indentation here.

Reply:
@martinmcconnell as there is a patch merge recently might be easier to you instead of rebase manually.